gpiocontrol/source/android/app/src/main/java/org/kaaproject/kaa/examples/gpiocontol/DevicesListActivity.java

```diff
@@ -1,41 +1,39 @@
 /**
```

Comment:
please, move all activities to some directory, for example ui/ Or you can use my approach and leave only one and main activity, activity which app loads. But two activities can't be in root directory